src/basic/filesystems.h

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include "stat-util.h"
+#include "nulstr-util.h"
+#include "string-util.h"
+
+#define FILESYSTEM_MAGIC_MAX 10
```

Comment:
isn't the enum enough?

Reply:
This is about the number of magic numbers per filesystem type in the `filesystems-gperf.gperf` table. For example for the "minix" name there are 5 magic numbers associated with it.